src/components/Dashboard/index.js

```diff
@@ -49,20 +60,31 @@ const Dashboard = () => {
   const [nameValidateErrorMessage, setNameValidateErrorMessage] = useState("");
   const [orgName, setOrgName] = useState("");
   const [orgNameValidateError, setOrgNameValidateError] = useState(false);
-  const [orgNameValidateErrorMessage, setOrgNameValidateErrorMessage] = useState("");
+  const [
+    orgNameValidateErrorMessage,
+    setOrgNameValidateErrorMessage,
+  ] = useState("");
```

Comment:
could be in a single line

Reply:
actually vscode prettier formating this lines . I'll edit this in different editor and commit those.